cisc/lib.go

```diff
@@ -94,18 +96,16 @@ func (cfg *ciscConfig) proposeSendVoteUpdate(p *identity.Config) {
 	log.ErrFatal(cfg.ConfigUpdate())
 }
 
-// writes the ssh-keys to an 'authorized_keys'-file
+// writes the ssh-keys to an 'authorized_keys.cisc'-file
 func (cfg *ciscConfig) writeAuthorizedKeys(c *cli.Context) {
 	var keys []string
 	dir, _ := sshDirConfig(c)
-	authFile := dir + "/authorized_keys"
-	// Make backup
-	b, err := ioutil.ReadFile(authFile)
-	if err == nil {
-		err = ioutil.WriteFile(authFile+".back", b, 0600)
-		log.ErrFatal(err)
+	authFileOrig := filepath.Join(dir, "authorized_keys")
```

Comment:
nice `filepath`. We use `path` in most of our code, we should migrate to using this one since it's apparently supporting windows..